src/containers/NonBitGoRecoveryCoin/ArbethTokenForm.tsx

```diff
@@ -0,0 +1,167 @@
+    formik.values.krsProvider === ''
+      ? 'Your encrypted backup key, as found on your recovery KeyCard.'
+      : 'The backup public key for the wallet, as found on your recovery KeyCard.';
+
+  return (
+    <FormikProvider value={formik}>
+      <Form>
+        <h4 className="tw-text-body tw-font-semibold tw-border-b-0.5 tw-border-solid tw-border-gray-700 tw-mb-4">
+          Self-managed hot wallet details
+        </h4>
+        <div className="tw-mb-4">
+          <FormikSelectfield
+            HelperText="The Key Recovery Service that you chose to manage your backup key. If you have the encrypted backup key, you may leave this blank."
+            Label="Key Recovery Service"
+            name="krsProvider"
+            Width="fill"
+          >
+            <option value="">None</option>
+            <option value="keyternal">Keyternal</option>
+            <option value="bitgoKRSv2">BitGo KRS</option>
+            <option value="dai">Coincover</option>
+          </FormikSelectfield>
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextarea
+            HelperText="Your encrypted user key, as found on your recovery KeyCard."
+            Label="Box A Value"
+            name="userKey"
+            rows={4}
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextarea
+            HelperText={backupKeyHelperText}
+            Label="Box B Value"
+            name="backupKey"
+            rows={4}
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The ARBETH address of the wallet contract. This is also the wallet's base address."
+            Label="Wallet Contract Address"
+            name="walletContractAddress"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The address of the smart contract of the token to recover. This is unique to each token, and is NOT your wallet address."
+            Label="Token Contract Address"
+            name="tokenContractAddress"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikPasswordfield
+            HelperText="The passphrase of the wallet."
+            Label="Wallet Passphrase"
+            name="walletPassphrase"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The address your recovery transaction will send to."
+            Label="Destination Address"
+            name="recoveryDestination"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="Gas limit for the ARBETH transaction. The value should be between 30,000 and 20,000,000. The default is 500,000 units of gas."
```

Comment:
Change the values as mentioned in unsigned sweep

Reply:
addressed